docs/source/modules/benchmark_tutorial.rst

```diff
@@ -338,6 +338,52 @@ For instance, the following is an excerpt from the
         assert ceiling.raw.median('split') == ceiling
         assert ceiling.uncorrected_consistencies.median('split') < ceiling
 
+Benchmark Card Creation
+-----------------------
+
+Please include a :code:`README.md` file along with your benchmark to aid users with understanding and implementation.
+As part of your README.md file, please include a YAML section using the following format as a guideline:
+
+.. code-block:: python
+
+      benchmark_details:
+        name: <name of the benchmark>
+        developer: <developing individual or organization>
+        date: <date of benchmark creation>
+        version: <version number>
+        type: <behavioral or neural>
+        description: <a short summary description>
+        license: <license details>
+        questions: <where to send questions>
+        citations: <citation information if relevant>
+
+      experiment:
+        task: <list of ArtificialSubject task values>
+        recording: <ArtificialSubject Recording type>
+        experiment_card: <reference any existing experiment cards>
+        bidirectionality: <unidirectional/bidirectional>
```

Comment:
this still makes no sense to me, it sounds like something specific to the model. What would a bidirectional experiment run on a human subject look like?

Reply:
modified benchmark_tutorial.rst to say:  

 bidirectionality: <(if relevant) unidirectional/bidirectional: whether bidirectionality was used to obtain e.g., internal recordings of the model>